theme/ItaliaTheme/_common.scss

```diff
@@ -16,6 +25,12 @@
 
   a.btn-tertiary {
```

Comment:
Non farei solo per i terziari, o c'è un motivo che mi sfugge?
Per esempio nella modale di ricerca che si apre cliccando la lente in alto a destra, il bottone Cerca sotto è un `a.btn-primary` quindi ha ancora il focus originale